src/components/projects/projects-table/projects-table/projects-table.tsx

```diff
@@ -24,21 +25,89 @@ interface DisplayProjectProps {
   allProjects: DisplayProject[];
 }
 
+/***
+ * Custom sorting order for wbsNums according to car, then project, then workPackage.
+ * @param a 1st wbsNum in string form
+ * @param b 2nd wbsNum in string form
+ * @param order Imported SortOrder values 'asc' or 'desc'
+ * @return number A number -1, 0, or 1 describing the value of a relative to b,
+ *                according to the specified SortOrder.
+ */
+function wbsNumSort(a: string, b: string, order: SortOrder): number {
+  const wbs_a = validateWBS(a);
+  const wbs_b = validateWBS(b);
+  if (wbs_a.car !== wbs_b.car) {
+    if (order === 'asc') {
+      return wbs_a.car > wbs_b.car ? 1 : -1;
```

Comment:
Does a sort function only return 0, 1, and -1? Or does a sort function return 0, positive number, and negative number?